RecoMET/METFilters/python/hcalLaserEventFilter_cfi.py

```diff
@@ -7,3 +7,10 @@
     badEvents=[]
 
 hcalLaserEventFilter = hcallaserevent.clone(BadRunEventNumbers=cms.vuint32(badEvents) )
+
+from Configuration.Eras.Modifier_run2_HCAL_2018_cff import run2_HCAL_2018
+run2_HCAL_2018.toModify( hcalLaserEventFilter,
+                           vetoByHBHEOccupancy = cms.bool( False),

```

Comment:
Please drop type specification: types were already defined, and removing it here may help catching possible mistypes